src/plugins/discover/public/services/doc_views/components/doc_viewer/doc_viewer.tsx

```diff
@@ -35,6 +35,7 @@ export function DocViewer(renderProps: DocViewRenderProps) {
             render={render}
           />
         ),
+        ['data-test-subj']: `docViewerTab-${title?.replace(/\s/g, '-')}`,
```

Comment:
I would suggest to use `idx` here instead of localized titles.

Reply:
I was debating whether to use `idx` or `title` for this. I'll update it to `idx` instead.